docs/source/tutorial.rst

```diff
@@ -100,10 +144,15 @@ And this is the result on Dataverse:
 filetree mode
 *************
 
+``--mode filetree`` is an export mode, i.e., it will mirror a snapshot of the current state of your :term:`DataLad dataset` to Dataverse.
+This is more human readable on Dataverse, but a clone
```

Comment:
I don't think you finished your thought here. "but a clone" ...?

Reply:
thanks for catching this!